pkg/printer/printer.go

```diff
@@ -94,6 +94,24 @@ func (p *Printer) PrintSummaryTable(summary Summary) {
 }
 
 func (p *Printer) printInColor(title string, color *color.Color) {
-	warningColorPrintFn := color.PrintfFunc()
-	warningColorPrintFn(title)
+	colorPrintFn := color.PrintfFunc()
+	colorPrintFn(title)
+}
+
+func (p *Printer) createNewColor(clr string) *color.Color {
+	switch clr {
+	case "red":
+		return p.theme.Colors.Red
+	case "yello":
+		return p.theme.Colors.Yello
+	case "green":
+		return p.theme.Colors.Green
+	default:
```

Comment:
we should also have case for explicit white color. in case we decide to change the default later on (backend supports white as well)

Reply:
what should be the default then?